apps/src/studio/bigGameLogic.js

```diff
@@ -69,6 +74,34 @@ BigGameLogic.prototype.onTick = function () {
       }
     }
   }
+  
+  if (this.collide(playerSprite.x, playerSprite.y, dangerSprite.x, dangerSprite.y) && playerSprite.visible && dangerSprite.visible) {
+    this.studio_.vanishActor({spriteIndex:this.playerSpriteIndex});
+    setTimeout((function() {this.studio_.setSprite({spriteIndex:this.playerSpriteIndex,value:"visible"});}).bind(this), 500);
+    this.studio_.playerScore -= 20;
```

Comment:
Couple of style nits:
(1) We try to limit lines to 80 characters.
(2) Space after :
(3) Split function out onto multiple lines
(4) When we have a hash with multiple items, we usually also split it out one per line
```
setTimeout(function () {
  this.studio._setSprite({
    spriteIndex: this.playerSpriteIndex,
    value: "visible"
  });
}).bind(this), 500);